nautobot/core/models/__init__.py

```diff
@@ -56,6 +56,7 @@ class BaseModel(models.Model):
 
     objects = BaseManager.from_queryset(RestrictedQuerySet)()
     is_contact_associable_model = True
+    is_saved_view_model = True
```

Comment:
I'm thinking that (much like Contacts and Static Groups) it might make more sense for `OrganizationalModel` and `PrimaryModel` to default to supporting saved views, and `BaseModel` to default to not supporting them.

Take a look at my latest diffs in #5631 to see how I'm using some mixins to implement that for those other two cases.